invenio_vocabularies/contrib/funders/mappings/os-v1/funders/funder-v2.0.0.json

```diff
@@ -0,0 +1,113 @@
+{
+  "settings": {
+    "analysis": {
+      "analyzer": {
+        "edge_analyzer": {
+          "tokenizer": "letter",
+          "filter": [
+            "lowercase",
+            "asciifolding",
+            "edgegrams"
+          ]
+        }
+      },
+      "filter": {
+        "edgegrams": {
+          "type": "edge_ngram",
+          "min_gram": 2,
+          "max_gram": 20,
+          "preserve_original": true
+        }
+      }
+    }
+  },
+  "mappings": {
+    "dynamic_templates": [
+      {
+        "i18n_title": {
+          "path_match": "title.*",
+          "match_mapping_type": "string",
+          "mapping": {
+            "type": "search_as_you_type"
+          }
```

Comment:
Shouldn't these be the same as in affiliations? The end-usage is the same, i.e. we'll be using the `/api/funders?suggest=...` calls in the UI for an autocomplete-style search input.

Reply:
Yes, thank you I missed this
Added!